doc/guide/feature-kubernetes.xml

```diff
@@ -9,13 +9,55 @@
     <ulink url="https://enterprise.openshift.com/">Openshift v3 cluster</ulink>. This
     functionality is in the Cockpit <emphasis>kubernetes</emphasis> package.</para>
 
-  <para>Cockpit communicates with Kubernetes via its REST API. In order to access this
-    API, usually some form of authentication is necessary. Like the <filename>kubectl</filename>
-    and <filename>oc</filename> commands, Cockpit uses the authentication and server
-    information in the <code>~/.kube/config</code> file for the logged in user.</para>
+  <para>The dashboard can either be deployed on kubernetes as a
+    <ulink url="http://kubernetes.io/docs/user-guide/pods/">container in a pod</ulink> or used
+    via a normal authenticated Cockpit session. Cockpit communicates with Kubernetes via its
+    REST API.</para>
 
-  <para>If a user is able to use <code>kubectl</code> successfully when at their shell terminal,
-    then that same user will able to use Kubernetes dashboard when logged into Cockpit:</para>
+  <section>
+    <title>Used as a Kubernetes Pod</title>
+
+    <para>The Kubernetes dashboard can be deployed as a pod on Kubernetes requiring users
+      to login using the standard standard authentication configured for access its API.</para>
+
+<programlisting>
+$ <command>wget https://github.com/cockpit-project/cockpit/tree/master/containers/kubernetes-cockpit.json</command>
```

Comment:
should this use the raw content?
https://raw.githubusercontent.com/cockpit-project/cockpit/master/containers/kubernetes-cockpit.json